src/AltinnCore/Common/Languages/ini/nb.ini

```diff
@@ -93,6 +93,11 @@ list_favourites = FAVORITTER
 modal_configure_address_component_simplified = Enkel
 modal_add_options_codelist = Kodeliste
 modal_add_options_manual = Manuelt
+modal_address = Adresse
+modal_address_zip_code = Postnr
+modal_address_post_place = Poststed
+modal_address_care_of = C/O eller annen tilleggsadresse
+modal_address_house_number = Bolignummer
```

Comment:
I have 1 text related to this advanced component, maybe have the same naming-prefixes as that one.

Reply:
Agreed, fixed.